api/apps/geoprocessing/src/modules/scenario-planning-units-inclusion/scenario-planning-units-inclusion-processor.ts

```diff
@@ -124,38 +122,19 @@ export class ScenarioPlanningUnitsInclusionProcessor
     const uniquePuIdsToExclude = new Set(puIdsToExclude);
     const uniquePuIdsToMakeAvailable = new Set(puIdsToMakeAvailable);
 
-    const doInclusionAndExclusionIntersect =
-      intersection([...uniquePuIdsToInclude], [...uniquePuIdsToExclude])
+    const doClaimsIntersect =
+      intersection([...uniquePuIdsToInclude], [...uniquePuIdsToExclude], [...uniquePuIdsToMakeAvailable])
```

Comment:
There seems to be an issue here, since when we are passing 3 arrays to intersection, it will return the part that exists in all 3 of them. So, for example, if include and exclude arrays have intersecting pus, but make available doesn't have same part - it will return empty array and no error will be thrown..
I suggest keeping same 3 intersection checks for each pair and unify error message like this:
``` if (
      doInclusionAndExclusionIntersect ||
      doMakeAvailableAndExclusionIntersect ||
      doMakeAvailableAndInclusionIntersect
    ) {
      throw new Error(
        'Contrasting claims to include, exclude or make available some of the planning units have been made: please check your selections.',
      );
    }```